docs/det_match.rst

```diff
@@ -0,0 +1,92 @@
+.. py:module:: sotodlib.coords.det_match
```

Comment:
Please add some autofunction and/or autoclass at the bottom of this file.  See tod_ops.rst for lots of examples

This not only dumps in the docstrings, directly, it also provides links to the source code (which is often what one ultimately needs, to understand the significance of some parameter / attribute).